Content.Server/GameObjects/EntitySystems/Click/InteractionSystem.cs

```diff
@@ -266,6 +267,16 @@ private void UserInteraction(IEntity player, GridCoordinates coordinates, Entity
                 return;
             }
 
+            // In a container where the attacked entity is not the container's owner and either the attacked entity is null, not contained or in a different container
+            if (ContainerHelpers.TryGetContainer(player, out var playerContainer) &&
+                attacked != playerContainer.Owner &&
+                (attacked == null ||
+                 !ContainerHelpers.TryGetContainer(attacked, out var attackedContainer) ||
+                 attackedContainer != playerContainer))
+            {
+                return;
+            }
+
```

Comment:
My god this is ugly. Maybe make a few consecutive `if` statements instead? With comments, of course.